contracts/interchain-token-service/src/contract/execute.rs

```diff
@@ -188,11 +187,19 @@ mod tests {
         let mut deps = init();
         let axl_denom = "uaxl";
         let eth_denom = "eth";
-        let token_id_axl = get_token_id(&deps.as_mut(), axl_denom).unwrap();
-        let token_id_eth = get_token_id(&deps.as_mut(), eth_denom).unwrap();
+        let token_id_axl = gateway_token_id(&deps.as_mut(), axl_denom).unwrap();
+        let token_id_eth = gateway_token_id(&deps.as_mut(), eth_denom).unwrap();
         assert_ne!(token_id_axl, token_id_eth);
     }
 
+    #[test]
+    fn get_token_id_should_not_change() {
```

Comment:
```suggestion
    fn gateway_token_id_should_not_change() {
```
update other test names as well